en/docs/learn/consume-api/discover-apis/search.md

```diff
@@ -74,6 +74,11 @@ You can search for APIs in the API Publisher or Developer Portal in the followin
 <p>A property can be defined to an API at the time it is created or later. There can be APIs without properties as this parameter is optional. <strong><br />
 </strong></p></td>
 </tr>
+<tr class="even">
+<td>By API category</td>
+<td><p><strong>api-category:xxxx</strong> . For example, api-category:test</p>
+<p>A category is used to group APIs based on any properties. API categories are defined in the Admin Portal and assigned in Design Configurations of the Publisher.</p></td>
```

Comment:
Shall we change this to the below sentence?
"An API category can be used to group APIs based on any property. API categories can be defined from the Admin Portal and can be assigned to an API in the Design Configurations section of the Publisher Portal."

Reply:
Makes sense :+1: 

Corrected https://github.com/wso2/docs-apim/pull/2615/commits/07830c88291acc22d63a264a647700db8ead2290